src/includes/getting-started-verify/javascript.svelte.mdx

```diff
@@ -0,0 +1,12 @@
+Add a button to a Svelte component that throws an error.
```

Comment:
This section already has a lead-in sentence for the code snippet in the common content (see screenshot below): 

![2022-08-15_14-10-43](https://user-images.githubusercontent.com/76437239/184691508-abfd8223-9b82-49c5-84c5-55690fc66c18.png)

So the lead-in text you've added doesn't make sense in this context. I can't make the suggestion directly here because the back ticks from the code snippet interfere, but I'd suggest that you remove the sentence about the button before the code snippet and add the following after the code snippet:

> This snippet adds a button that throws an error to a Svelte component.

Reply:
Makes sense. I think we have the same problem in the Remix SDK docs (which I took as an example). Will open another PR to adjust this (among some other things I came across)